src/atoms/Flex/Flex.web.js

```diff
@@ -1,3 +1,76 @@
-const Flex = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Flex = styled(
+  ({
+    alignContent,
+    alignItems,
+    alignSelf,
+    display,
+    flex,
+    flexBasis,
+    flexDirection,
+    flexGrow,
+    flexShrink,
+    flexWrap,
+    justifyContent,
+    order,
+    children,
+    style,
+    ...props
+  }) =>
+    React.cloneElement(children, {
+      ...props,
+      style: { ...style, ...children.props.style },
+    }),
+)`
+  &&& {
```

Comment:
Why do you need ```&&& { .... }```

Can't we just write it the way it's on ```Flex.native.js```?

Reply:
Excellent Question! We can't because Css specificity issues my boi. This basically increases the specificity so no other styles can override this unless explicitly mentioned using `!important`. [Read Here](https://razorpay.slack.com/archives/CTKFJ1QVC/p1585041853007200)

Reply:
Nice! :O

But do we need it here? Won't it cause confusion while debugging as to why some styles might be getting overwritten? The consumer will then have to dive into the code of our DS and look into this specific place to understand the higher specificity?

Doing this on a project makes sense where it is _our_ code but on a DS it seems like a heavy enforcement, no?

Reply:
No it’s basically to ensure that our enhancers space, flex etc work as expected it’s not breaking anything in the consumer codebase it’s just applying the class with higher specificity for this particular style not everything